### DISCUSSION TITLE
Locking dark mode doesn't use normal hover styles

### DISCUSSION THREAD

**Discussion by codergautam** [tailwindlabs/tailwindcss#11732] [Help]
We have been building an app with Tailwind that supports both light and dark mode. The dark mode option in the config is set to the default `media`. It works fine this way.

Since our deadline is approaching fast we're trying to make testing easier by "locking" to a specific mode. We achieved this by adding `<html class="dark>` and setting the darkMode value to `class`. This worked, but there's one annoying issue. The hover styles and other styles we specified without the dark: selector don't work. 

I was assuming that dark mode would just prioritize the dark: selector, not just disable the other ones completely. Not to mention that this is how it is when dark mode is set to media, so not sure why it does this when I try to lock to dark mode. Is there any other way of locking dark mode without having to mess with our old styles and keeping the functionality same as how it was in media and browser mode set to dark?

Thanks

**Comment by wongjn** [comment]
Consider providing a [Tailwind Play](https://play.tailwindcss.com) that reproduces the unexpected behavior.
